docs/components/search-bar/index.html

```diff
@@ -0,0 +1,31 @@
+--- 
+  title: Menus 
+  also: 
+    elements/hx-menu: <hx-menu>
+    elements/hx-menuitem-group: <hx-menuitem-group>
+    elements/hx-menuitem: <hx-menuitem>
+--- 
+{% extends 'component.njk' %} 
+{% block content %}
+  <section id="vue-searchBarDemo">
+    <h2 class="hxSectionTitle" id="search-bar">Basic Search Bar</h2>
+    <p>
+      State:
+      <select v-model="state">
+        <option :value="_state" v-for="_state in states" v-text="_state"></option>
+      </select>
+    </p>
+    <div class="demo">
+      <hx-search-bar :error="state === 'error' ? true : false" :empty="state === 'empty' ? true : false">
```

Comment:
The custom element can be renamed as `<hx-search>` to shorten the name.

Reply:
👍